### DISCUSSION TITLE
Using POST timer

### DISCUSSION THREAD

**Discussion by ICULikeMac** [babybuddy/babybuddy#459] [Support] [answered]
Hi all!

Is someone able to clarify what is supposed to be in the 'value' field if I add a 'timer' to the key/index of a /api/sleep/ request?

So I want to replace the start / stop to use a timer? {"child":1,"start":"2022-05-28T14:00","end":"2022-05-28T15:00"}

I have added a bool, "timer": true, however, then I get a response {"timer":["Timer does not exist."]}.

Thank you!!

**Comment by cdubz** [comment]
@ICULikeMac The `timer` value needs to be a timer _ID_ (not a `bool`, as there could be more than one timer). The documentation is a little unclear on this, for sure: https://docs.baby-buddy.net/api/#timer-field

**Comment by cdubz** [comment, marked as answer]
@ICULikeMac FYI I have updated the timer field documentation with clarification and an example usage: http://docs.baby-buddy.net/api/#timer-field

**Reply by ICULikeMac** [reply]
@cdubz That's awesome, thanks very much! Love this project, good work and thanks again. 

I'm doing up an iOS app, if it ends up any good (And the bub doesn't arrive too early), I'll share it.

**Reply by cdubz** [reply]
Neat! Please do share when you’re ready. Good luck! 👶

**Reply by ICULikeMac** [reply]
Will do - Thank you!
